app/src/main/java/sh/siava/AOSPMods/SettingsActivity.java

```diff
@@ -263,6 +263,7 @@ private void importExportSettings(boolean export) {
 		Intent fileIntent = new Intent();
 		fileIntent.setAction(export ? Intent.ACTION_CREATE_DOCUMENT : Intent.ACTION_GET_CONTENT);
 		fileIntent.setType("*/*");
+		fileIntent.putExtra(Intent.EXTRA_TITLE, "AOSPMods_Config" + ".bin");
```

Comment:
Proprietary file extensions are not a good idea, and .bin can represent config files well

Reply:
bin is very generic binary file as for myself. config files can have lots of other extensions as xml, cnf, cfg or even conf. imho they might look better than generic .bin. What do you think?

Reply:
btw, .asf is a video format

Reply:
> bin is very generic binary file as for myself. config files can have lots of other extensions as xml, cnf, cfg or even conf. imho they might look better than generic .bin. What do you think?

if ur config file can be edit,I will set as cfg or xml,etc.
but I check with the file, it seems like binary to me, so I set to .bin as it doesn't seems include structure

Reply:
![image](https://github.com/siavash79/AOSPMods/assets/29218242/d75d3ff7-0e94-47a6-85fb-f57a8bc86626)
It does not seem to be editable to me.

Reply:
It is a hashmap file. Not intended for editing at all. For storing the configuration. BUT you can also edit it in case you want (if you understand what is what)

![IMG_20230514_182950.png](https://github.com/siavash79/AOSPMods/assets/27622683/26bd97d2-b77e-4113-8dfe-5bc4eb1b865d)

ASF can be video format. Thst is why I advised conf or cnf extension. They are also not intended for editing purposes. Though they can be edited by users who understand what is what

Reply:
> 
 conf, cnf, xml, txt,they all represent files that can be edited or design to be edited
bin or hex is what it should be as a hashmap